server/lib/endpoints/northstar/users.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const OAuthEndpoint = require('../oauth-endpoint');
+
+/**
+ * @see https://github.com/DoSomething/northstar/blob/master/documentation/endpoints/v2/users.md
+ */
+class NorthstarEndpointUsers extends OAuthEndpoint {
+  constructor(client) {
+    super(client);
+    this.url = `${this.client.config.services.northstar.baseUri}/users`;
+  }
+  /**
+   * @param  {String} id
+   * @param  {Object} query
+   * @return {Promise}
+   */
+  get(id, query) {
+    return this
+      .executeGet(`${this.url}/${id}`, query)
+      .then(responseBody => responseBody);
```

Comment:
What's the `responseBody => responseBody` step on each of these doing?

Reply:
Hmm, doesn't seem to be doing much. It's copy/paste code from https://github.com/DoSomething/gateway-js/pull/20. I think it'd be cleaner to remove it, although I could guess the intent may be to clarify that we're sending the `res.body` back per parsing in the `executeGet` function. I'm into removing these ✂️ 

It's funny, my suggestions in that PR https://github.com/DoSomething/gateway-js/pull/20 to @rapala61 mentioned renaming `get` to `getById` , but it was also because the class had other `getActivityByStuff` type of functions. It seemed most apps could handle their own helpers to write queries .. and per https://github.com/DoSomething/gateway-js/pull/21#discussion_r240409677 - this is also where I started wondering about the various endpoints in this `server/src` are duplicating [GraphQL code](https://github.com/DoSomething/graphql/tree/master/src/repositories).